spatial/r3/rotation.go

```diff
@@ -0,0 +1,43 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
```

Comment:
I think this belongs in vector.go.

Reply:
you mean move everything that is inside `rotation.go` over to `vector.go`?

Reply:
Yep